app/src/main/java/org/schabi/newpipe/player/resolver/PlaybackResolver.java

```diff
@@ -43,13 +44,13 @@ default MediaSource buildLiveMediaSource(@NonNull final PlayerDataSource dataSou
         switch (type) {
             case C.TYPE_SS:
                 return dataSource.getLiveSsMediaSourceFactory().setTag(metadata)
-                        .createMediaSource(uri);
+                        .createMediaSource(new MediaItem.Builder().setUri(uri).build());
```

Comment:
These MediaItem builders can be simplified to just `MediaItem.fromUri(uri)`

Reply:
you are right. I was copying that over from Exoplayer sources even though I should have known better, fixed. Thanks